src/decodePointer.ts

```diff
@@ -1,10 +1,27 @@
 import { replaceInString } from './_utils';
 
+function safeDecodeURIComponent(value: string): string {
+  try {
+    return decodeURIComponent(value);
+  } catch {
+    return value;
```

Comment:
This is veeeery unlikely to be every encountered, but I added it just in case.

Reply:
LMFAO